back/tests/openbsrv/postsvc_test.go

```diff
@@ -295,6 +296,75 @@ func postSvcFindAllPosts(ctx context.Context, conn *grpc.ClientConn, clnt pb.Pos
 				t.Errorf("unexpected error: %v", err)
 			}
 
+			if len(tc.want.Posts) != len(resp.Posts) {
+				t.Errorf("mismatch between response length and post length, "+
+					"want: %d got: %d", len(tc.want.Posts), len(resp.Posts))
+			}
+
+			for _, post := range tc.want.Posts {
+				if !postsContain(resp, post) {
+					t.Errorf("couldn't find post with title: %s", post.Title)
+				}
+			}
+		}
+	}
+}
+
+func postSvcKeywordSearch(ctx context.Context, conn *grpc.ClientConn, clnt pb.PostClient) func(*testing.T) {
+	return func(t *testing.T) {
+		tests := []struct {
+			fndReq *pb.FndPostsReq
+			want   *pb.PostsResp
+		}{
```

Comment:
I'm not sure if we've been following this practice consistently so far... I suggest adding a field for the name of the test. The name is useful to filter for specific tests (when tests are eventually organized to be run independently), and it also documents the nature/intent of the test. For myself, thinking about the nature/name of a test case often brings to mind other test cases that would make sense to nail down.